mlprimitives/timeseries.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+import numpy as np
+
+
+def create_window_sequences(df_timeseries, window_size):
+    """
+        Function that takes in a Pandas.DataFrame and a window_size then creates
+            output arrays that correspond to a timeseries sequence with window_size overlap.
+            The output arrays can be fed into a timeseries forecasting model.
+        Inputs:
```

Comment:
Please follow the google docstring style: `Args` instead of `Inputs` and `Returns` instead of `Outputs`.
Also, in the `Returns` section you only specify the type, not the name.